src/terminal/init.lua

```diff
@@ -331,6 +340,10 @@ end
 -- @treturn string ansi sequence to write to the terminal
 -- @within cursor_position
 function M.cursor_sets(row, column)
+ -- Retrieve terminal dimensions once
+local size = sys.get_terminal_size()
+row = utils.resolve_index(row, size.height)
+column = utils.resolve_index(column, size.width)
```

Comment:
```suggestion
  -- Retrieve terminal dimensions once
  local size = sys.get_terminal_size()
  row = utils.resolve_index(row, size.height)
  column = utils.resolve_index(column, size.width)
```
indentation is of a little bit here